testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -267,7 +267,7 @@ TestNodeProxy : UnitTest {
 		timeout.stop;
 		resp.free;
 
-		this.assertEquals(result, proxy.source, "after the crossfade from a ugen function to a value the bus should have this value");
+		this.assertFloatEquals(result, proxy.source, "after the crossfade from a ugen function to a value the bus should have this value");
```

Comment:
Did `test_control_fade` fail before? If not, what's the rationale for this change?